int32 dbot_h
int32 dbot_roll
int32 dbot_p
int32 dbot_m
int32 dbot_ctl
int32 dbot_act